gym_bot_app/commands/month_ranking.py

```diff
@@ -0,0 +1,77 @@
+from telegram import Update
+from telegram.ext import CallbackContext
+from datetime import datetime
+from calendar import month_name
+
+
+from gym_bot_app.decorators import get_group
+from gym_bot_app.commands import Command
+from gym_bot_app.models import Group
+
+
+class MonthRankingCommand(Command):
+    """Telegram gym bot month ranking command.
+
+    Sends group month ranking based on trainees statistics.
+
+    """
+    DEFAULT_COMMAND_NAME = 'month_ranking'
+    TRAINEES_LIMIT = 10
+    DID_NOT_PROVIDE_LEGAL_MONTH = 'לא הכנסת חדוש תקין יא בוט'
+
+    @get_group
+    def _handler(self, update: Update, context: CallbackContext, group: Group):
+        """Override method to handle month ranking command.
+
+        Takes top trainees based on their average monthly training.
+
+        """
+        self.logger.info('Ranking statistics command in %s', group)
+
+        month = 0
+        if len(context.args) == 0:
+            self.logger.debug('Trainee did not provide month')
+            month = datetime.now().month
+
+        elif not any(chr.isdigit() for chr in context.args[0]):
+            self.logger.debug('Trainee did not provide legal month')
+            update.message.reply_text(quote=True, text=self.DID_NOT_PROVIDE_LEGAL_MONTH)
+            return
+        else:
+            month = int(context.args[0])
+
+        if month <= 0 or month > 12:
+            self.logger.debug('Trainee did not provide legal month')
+            update.message.reply_text(
+                quote=True, text=self.DID_NOT_PROVIDE_LEGAL_MONTH)
+            return
+
+        trainees = [self._get_trainee_properties(trainee, month) for trainee in group.trainees]
```

Comment:
Better split the code and use kwargs like this
```suggestion
        trainees_properties = [
            self._get_trainee_properties(trainee=trainee, month=month) 
            for trainee in group.trainees
        ]
```

Reply:
d